openapi/openapi.yaml

```diff
@@ -819,12 +799,24 @@ components:
           type: object
           properties:
             overall:
-              type: array
+              type: object
               description: metrics for each analysis level
-              items:
-                type: array
-                items:
-                  $ref: "#/components/schemas/Performance"
+              additionalProperties:
+                description: analysis level
+                type: object
+                additionalProperties:
+                  description: metric name
+                  $ref: "#/components/schemas/MetricResult"
+              example: {
+                "example": {
+                  "rouge2": {
+                    "values": {
+                      "score": {"value": 0.12},
+                      "score_ci": {"low": -0.07, "high": 0.34, "alpha": 0.05}
+                    }
+                  }
+                }
+              }
```

Comment:
This may break the `explainaboard_client` CLI: https://github.com/neulab/explainaboard_client/blob/e4ec42991d22327b69952891a8b5750fe27d6c4c/explainaboard_client/cli/evaluate_system.py#L159

If it does we should open an issue there before merging this PR.

Reply:
Thanks for the catch! Let me work on this and submit a PR along with the Benchmark CRUD and system tag support. ETA end of today.

Reply:
Sorry please ignore my previous confusing comment. I've opened an issue here: https://github.com/neulab/explainaboard_client/issues/58 

After this PR is merged, I'll work on it before upgrading to the latest API client.

Reply:
Thanks for the discussion! I tested `explainaboard_client` and I don't think this PR would affect that project. I only changed `SystemInfo` (the object we cache) in this PR which is only used in `POST /systems/analyses`. The client uses `GET /systems` and `POST /systems` instead so it should still work.